packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -183,9 +192,13 @@ const Children = createComponentSlot(MuiAccordionDetails)<FieldSetClassKey, Owne
     ({ theme, ownerState }) => css`
         display: flex;
         flex-direction: column;
-        border-top: 1px solid ${theme.palette.divider};
         padding: 20px;
 
+        ${!ownerState.hiddenSummary &&
+        css`
+            border-top: 1px solid ${theme.palette.divider};
+        `}
```

Comment:
Wouldn't it be easier if the summary took care of the divider (by using border-bottom)? This way, no additional owner state and class key would be required.

Reply:
I thought of that but this would have just moved that problem to the summary, as the border then needs to be hidden when the `FieldSet` is collapsed.